docs/reference/api/README.md

```diff
@@ -82,6 +79,24 @@ To generate a version of the API:
    After a successful build, the tool automatically opens that folder in the
    `tmp` directory.
 
+   If the script fails, there are a couple possible causes.
+
+   * If you get the
+     `F1116 15:21:23.549503   63473 main.go:129] no API packages found in ./pkg/apis`
+     error, check if a new version of the script is available:
+      https://github.com/ahmetb/gen-crd-api-reference-docs/tags
+
+      The script is kept up-to-date with changes that go into the Kubernetes API.
+      As Knative adds support for those APIs, you might need make sure the
+      corresponding
+      [script `gen-crd-api-reference-docs` version](https://github.com/knative/docs/blob/master/hack/gen-api-reference-docs.sh#L26)
+      is used.
+
+   * If you get the
+     `F0807 13:58:20.621526 168834 main.go:444] type invalid type has kind=Unsupported which is unhandled`
+     error, the import target might have moved. For more details and an example,
```

Comment:
```suggestion
     error, the import target might have moved. For more details, and an example,
```
I'm bad at punctuation, maybe the above update looks correct ^

Reply:
Thanks Mike! Here is the style guide we use for Knative: https://developers.google.com/style/commas

Reply:
I've been really busy this week and decided to add the details about the issue i faced this week and quickly dumped details about an issue from the past. I took a second pass at this to better explain. PTAL